osu.Game.Rulesets.Catch/Skinning/Argon/ArgonBananaPiece.cs

```diff
@@ -78,5 +91,32 @@ private void load()
                 }
             });
         }
+
+        protected override void Update()
+        {
+            base.Update();
+
+            const float parent_scale_application = 0.4f;
+
+            // relative to time on screen
+            const float lens_flare_start = 0.3f;
+            const float lens_flare_end = 0.3f;
+
+            // Undo some of the parent scale being applied to make the lens flare feel a bit better..
+            float scale = parent_scale_application + (1 - parent_scale_application) * (1 / (ObjectState.DisplaySize.X / (CatchHitObject.OBJECT_RADIUS * 2)));
+
+            stabilisedPieceContainer.Rotation = -ObjectState.DisplayRotation;
+            stabilisedPieceContainer.Scale = new Vector2(scale, 1);
+
+            double duration = ObjectState.HitObject.StartTime - ObjectState.DisplayStartTime;
+
+            fadeContent.Alpha = MathHelper.Clamp(
+                Interpolation.ValueAt(
+                    Time.Current, 1f, 0f,
+                    ObjectState.DisplayStartTime + duration * lens_flare_start,
+                    ObjectState.DisplayStartTime + duration * lens_flare_end,
```

Comment:
Both `lens_flare_start` and `lens_flare_end` have the same value of `0.3f`, so this appears to be the same value twice? What's going on here?

Reply:
Well spotted. Huge oversight in final clean-ups.